tests/views_test.py

```diff
@@ -0,0 +1,57 @@
+from todo_app.data.itemStatus import ItemStatus
+from todo_app.data.todo_item import TodoItem
+from todo_app.views.view_model import ViewModel
+
+
+def test_return_done_items_only():
+    items = [
+        TodoItem('1', 'task 1', ItemStatus.TODO.value,
+                 'description of task 1'),
+        TodoItem('2', 'task 2', ItemStatus.TODO.value, ''),
+        TodoItem('3', 'task 3', ItemStatus.PROGRESS.value, ''),
+        TodoItem('4', 'task 4', ItemStatus.DONE.value, 'optional property')
+    ]
+    item_view_model = ViewModel(items)
+
+    # Get the expected list of items with 'Done' status
+    expected_done_items = [
+        item for item in items if item.status == ItemStatus.DONE.value]
+
+    assert item_view_model.done_items == expected_done_items
+    assert len(expected_done_items) == 1
+
+
+def test_return_in_progress_items_only():
+    items = [
+        TodoItem('1', 'task 1', ItemStatus.TODO.value,
+                 'description of task 1'),
+        TodoItem('2', 'task 2', ItemStatus.PROGRESS.value, ''),
+        TodoItem('3', 'task 3', ItemStatus.PROGRESS.value, ''),
+        TodoItem('4', 'task 4', ItemStatus.DONE.value, 'optional property')
+    ]
+    item_view_model = ViewModel(items)
```

Comment:
Minor, but I note that this view model is a carbon copy - would it be better directly shared as a fixture? E.g. we could have something like:
```python
@pytest.fixture
def item_view_model():
    items = [
        TodoItem('1', 'task 1', ItemStatus.TODO.value,
                 'description of task 1'),
        TodoItem('2', 'task 2', ItemStatus.TODO.value, ''),
        TodoItem('3', 'task 3', ItemStatus.PROGRESS.value, ''),
        TodoItem('4', 'task 4', ItemStatus.DONE.value, 'optional property')
    ]
    return ViewModel(items)

def test_return_done_items_only(item_view_model):
    done_items = item_view_model.done_items

    # ... continue with assertions from here
```

This way we'd cut out a chunk of the code in each test so they can be shorter and easier to digest